### DISCUSSION TITLE
Q: 前台只显示最新评论，以前的评论突然不显示了，但后台数据未被删除，有办法显示全部评论吗，还是只是单纯bug了？ || Q: Only the latest comments are displayed in the front desk. Previous comments suddenly disappear, but the background data has not been deleted. Is there a way to display all comments? Or is it just a bug? Already?

### DISCUSSION THREAD

**Discussion by lucays** [walinejs/waline#2134] [Q&A] [answered]
### Describe the bug | 问题描述

今天发现，网站上只有最新的几条评论了，但是看管理页面是有全部评论的，leancloud也有所有评论都没被删除，网站上只显示这两天的评论，之前的都不显示了。
翻文档没找到对应的机制说明，简单看了下源码recent comment似乎只是按count和page分页，跟时间之类的没啥关系。
如果是特殊机制的话，是什么地方限制了连10月24日的评论都不再显示了呢？

看log，前台显示页面走的是/comment接口并且标[Deprecated]，不需要token
/ui的管理页面会正常显示全部评论，走的是/api/comment接口就是正常的，但是需要token
![image](https://github.com/walinejs/waline/assets/29814851/33783a37-1aa8-426c-a458-4b7c022dfc89)
![image](https://github.com/walinejs/waline/assets/29814851/5d030019-5d14-488c-825a-d5419d242945)

部署在netlify是今天发现这个问题后才从vercel切到netlify的，切换前和切换后问题都仍然存在，应该跟这个没关系。

### Website URL | 问题网站

https://ocg-rule.readthedocs.io/zh-cn/latest/c06/2023.html#id228

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Today I discovered that there are only the latest few comments on the website, but if you look at the management page, all the comments are there, and all comments on leancloud have not been deleted. Only the comments from the past two days are displayed on the website, and none of the previous ones are displayed.
I couldn't find the corresponding mechanism description after looking through the document. After briefly looking at the source code, recent comment seems to be just paging by count and page, and has nothing to do with time or anything like that.
If it is a special mechanism, what is the restriction that even the comments on October 24 will no longer be displayed?

Looking at the log, the front-end display page uses the /comment interface and is marked [Deprecated], and no token is required.
The management page of /ui will display all comments normally. It is normal to use the /api/comment interface, but a token is required.
![image](https://github.com/walinejs/waline/assets/29814851/33783a37-1aa8-426c-a458-4b7c022dfc89)
![image](https://github.com/walinejs/waline/assets/29814851/5d030019-5d14-488c-825a-d5419d242945)

The deployment in netlify was switched from vercel to netlify after discovering this problem today. The problem still exists before and after the switch. It should not be related to this.

### Website URL | Problem website

https://ocg-rule.readthedocs.io/zh-cn/latest/c06/2023.html#id228

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
评论对应的文章页面地址不一样，被认为是两个页面的评论了。这个是评论和页面关联的 id。

![image](https://github.com/walinejs/waline/assets/424491/d06036c1-91de-4c49-b990-f7a6bf37cd76)


---
The article page addresses corresponding to the comments are different and are considered to be comments on two pages. This is the id associated with the comment and the page.

![image](https://github.com/walinejs/waline/assets/424491/d06036c1-91de-4c49-b990-f7a6bf37cd76)

**Reply by lucays** [reply]
噢，原来如此，感谢。原来是readthedocs修改了语言相关的url。。。

---
Oh, that’s right, thanks. It turns out that readthedocs modified the language-related URL. . .
